apps/src/maze/maze.js

```diff
@@ -259,7 +259,7 @@ function drawMap () {
         obsIcon.setAttribute('height', Maze.MARKER_HEIGHT * skin.obstacleScale);
         obsIcon.setAttribute('width', Maze.MARKER_WIDTH * skin.obstacleScale);
         obsIcon.setAttributeNS(
-          'http://www.w3.org/1999/xlink', 'xlink:href', skin.obstacle);
+          'http://www.w3.org/1999/xlink', 'xlink:href', skin.obstacleIdle || skin.obstacle);
```

Comment:
Again would prefer to stick to one, and make sure it gets set appropriately on the skin.